flink-runtime/src/test/java/org/apache/flink/runtime/testutils/ZooKeeperTestUtils.java

```diff
@@ -81,10 +81,6 @@ public static Configuration configureZooKeeperHA(
 		config.setString(CheckpointingOptions.CHECKPOINTS_DIRECTORY, fsStateHandlePath + "/checkpoints");
 		config.setString(HighAvailabilityOptions.HA_STORAGE_PATH, fsStateHandlePath + "/recovery");
 
-		// Akka failure detection and execution retries
-		config.setString(AkkaOptions.WATCH_HEARTBEAT_INTERVAL, "1000 ms");
```

Comment:
I suppose the idea behind removing them, instead of an update to the new options, is that since they didn't fail with ineffective options they obviously don't needed them in the first place? (same question for kafka)

Reply:
Yes exactly. They weren't used for several releases now.